sys/include/net/ieee802154_security.h

```diff
@@ -101,14 +101,14 @@ struct ieee802154_sec_dev {
     void *ctx;
 };
 
-#if !defined(IEEE802154_SEC_DEFAULT_KEY) || defined(DOXYGEN)
+#if !defined(CONFIG_IEEE802154_SEC_DEFAULT_KEY) || defined(DOXYGEN)
 /**
  * @brief   AES key that is used in the test vectors from the specification
  *
  * @note    Predefine it yourself,
  *          if you want another key to be set up on initialization
  */
-#define IEEE802154_SEC_DEFAULT_KEY              { 0xc0, 0xc1, 0xc2, 0xc3,   \
+#define CONFIG_IEEE802154_SEC_DEFAULT_KEY       { 0xc0, 0xc1, 0xc2, 0xc3,   \
```

Comment:
Could we have the same default value in both the macro and Kconfig version?

Reply:
Sure. Do you know if Kconfig has a byte array type? I did not find that type when I searched on google so I made it a string.
Or should I make `CONFIG_IEEE802154_SEC_DEFAULT_KEY` macro a string as well?

Reply:
> Sure. Do you know if Kconfig has a byte array type? I did not find that type when I searched on google so I made it a string.

No, it does not have such a type.

> Or should I make `CONFIG_IEEE802154_SEC_DEFAULT_KEY` macro a string as well?

I see no reason not to make it a string as well